README.md

```diff
@@ -67,7 +67,6 @@ An alternative to the Grunt build is provided via the `util.sh` shell script.
 
 ## Dependency Setup
 ----
-1.  `pushd .`
```

Comment:
No love for pushd or popd? (would love these back as they preserve your shell state in case of copy/paste)